prbench-imitation-learning/scripts/lerobot_eval_multi_seed.py

```diff
@@ -0,0 +1,311 @@
+            )
+        else:
+            # Fallback to overall average if per-task data not available
+            logging.warning(
+                "Could not extract per-episode data, using overall avg_sum_reward"
+            )
+            avg_reward_successful = overall_info.get("avg_sum_reward", 0.0)
+
+        # Store results for this seed
+        seed_result = {
+            "seed": current_seed,
+            "success_rate": float(success_rate),
+            "avg_sum_rewards_successful": float(avg_reward_successful),
+            "wall_clock_time_per_episode": float(time_per_episode),
+            "total_wall_clock_time": float(total_time),
+            "n_episodes": cfg.eval.n_episodes,
+            "full_info": info,
+        }
+        all_results["seeds"].append(seed_result)  # type: ignore
+
+        # Track for aggregation
+        all_success_rates.append(success_rate)
+        all_avg_rewards_successful.append(avg_reward_successful)
+        all_wall_clock_times.append(time_per_episode)
+
+        # Print seed results
+        logging.info(
+            colored(f"\nResults for seed {current_seed}:", "green", attrs=["bold"])
+        )
+        logging.info(f"  Success Rate: {success_rate:.4f}")
+        logging.info(f"  Avg Sum Rewards (Successful): {avg_reward_successful:.4f}")
+        logging.info(f"  Wall-clock Time per Episode: {time_per_episode:.4f} seconds")
+        logging.info(f"  Total Time: {total_time:.2f} seconds")
+
+    # Compute aggregated metrics across all seeds
+    all_results["aggregated_metrics"] = {
+        "success_rate": {
+            "mean": float(np.mean(all_success_rates)),
+            "std": float(np.std(all_success_rates)),
+            "min": float(np.min(all_success_rates)),
+            "max": float(np.max(all_success_rates)),
+            "values": [float(x) for x in all_success_rates],
+        },
+        "avg_sum_rewards_successful": {
+            "mean": float(np.mean(all_avg_rewards_successful)),
+            "std": float(np.std(all_avg_rewards_successful)),
+            "min": float(np.min(all_avg_rewards_successful)),
+            "max": float(np.max(all_avg_rewards_successful)),
+            "values": [float(x) for x in all_avg_rewards_successful],
+        },
+        "wall_clock_time_per_episode": {
+            "mean": float(np.mean(all_wall_clock_times)),
+            "std": float(np.std(all_wall_clock_times)),
+            "min": float(np.min(all_wall_clock_times)),
+            "max": float(np.max(all_wall_clock_times)),
+            "values": [float(x) for x in all_wall_clock_times],
+        },
+    }
+
+    # Print final aggregated results
+    logging.info(
+        colored(
+            f"\n{'='*80}\nAggregated Results Across {cfg.num_seeds} Seeds\n{'='*80}",
+            "cyan",
+            attrs=["bold"],
+        )
+    )
+    logging.info(colored("\nSuccess Rate:", "green", attrs=["bold"]))
+    logging.info(
```

Comment:
to avoid all these `# type: ignore # pylint: disable=line-too-long`, you could try

```python
metrics = all_results['aggregated_metrics']
```